From: Julian Ospald <hasufell@gentoo.org>
Date: Wed Aug 27 02:14:46 UTC 2014
Subject: add dbus switch

	https://github.com/notsecure/uTox/pull/316

--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,18 @@
+DBUS=1
+
 UNAME_S := $(shell uname -s)
 
 CFLAGS += $(shell pkg-config --cflags freetype2 x11 openal)
+ifeq ($(DBUS), 1)
 CFLAGS += $(shell pkg-config --cflags dbus-1)
+else
+CFLAGS+=-DNO_DBUS
+endif
 CFLAGS += -g -pthread -std=gnu99
 LDFLAGS += $(shell pkg-config --libs freetype2 x11 openal)
+ifeq ($(DBUS), 1)
 LDFLAGS += $(shell pkg-config --libs dbus-1)
+endif
 LDFLAGS += -lX11 -lXft -lXrender -ltoxcore -ltoxav -ltoxdns -lopenal -pthread -lm -lfontconfig -lv4lconvert -lvpx -lXext
 
 ifeq ($(UNAME_S),Linux)
